src/server/system_services/account_server.js

```diff
@@ -607,12 +612,13 @@ function list_accounts(req) {
         accounts = system_store.data.accounts;
     } else if (req.account) {
         // list system accounts - system admin can see all the system accounts
-        if (!_.includes(req.account.roles_by_system[req.system._id], 'admin')) {
-            throw new RpcError('UNAUTHORIZED', 'Must be system admin');
+        if (!_.includes(req.account.roles_by_system[req.system._id], 'admin') &&
```

Comment:
@guymguym why do we actually need this check here? can't we just rely on the auth?